master/internal/db/postgres_experiments_intg_test.go

```diff
@@ -54,7 +62,29 @@ func TestExperimentCheckpointsToGCRaw(t *testing.T) {
 		0,
 	)
 	require.NoError(t, err)
-	require.Equal(t, expectedCheckpoints, checkpoints)
+	sort.Slice(expectedCheckpoints1, func(i, j int) bool {
```

Comment:
could you split this into two tests instead or, even better, use table driven tests (https://dave.cheney.net/2019/05/07/prefer-table-driven-tests)? as a rule of thumb, one test should test one thing; it is much easier to maintain the tests then (especially understand them when failures happen).